ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -262,9 +262,7 @@ service_env:
   discussionsmw: ../../../../ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env
 
   dhiti: ../../../../ansible/roles/stack-sunbird/templates/dhiti-service.env
-  dial:
-    - ../../../../ansible/roles/stack-sunbird/templates/dial-service_application.conf
-    - ../../../../ansible/roles/stack-sunbird/templates/dial-service_logback.xml
+  sl-assessments: ../../../../ansible/roles/stack-sunbird/templates/sl-assessments-service.env
```

Comment:
@sivakarthikeyanoptit why's the dial service got deleted ?

Reply:
dial while KT raj asked me to remove, if it is required i will add it

Reply:
dude, why did you mark it as resolved? why you had to delete the dial service dict ?

Reply:
i have seen previous merged file as reference rajesh, you are correct it is there, i have included the dial service

Reply:
done